pkgs/tools/security/ffuf/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "ffuf";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "ffuf";
+    repo = "ffuf";
+    rev = "v${version}";
+    sha256 = "1jb2x0ybcb9zkqm7flpmr0hd3171xvnn6kxmfcgds4x8l9fbmxnr";
+  };
+
+  vendorSha256 = "0sjjj9z1dhilhpc8pq4154czrb79z9cm044jvn75kxcjv6v5l2m5";
+
+  doCheck = false;
```

Comment:
Can you comment about why the tests fail?

Reply:
oh, copy-paste fail. there are no tests, there are no failures.

Reply:
ok, there are test failures after all. Investigating.

Reply:
@htr you don't have to comment exactly why the tests fail - you can write something simple as `# tests fail`.